MCprep_addon/mcprep_ui.py

```diff
@@ -609,6 +627,28 @@ def draw(self, context):
 			col = split.column()
 			p = col.operator("mcprep.openfolder", text=env._("Open texture pack folder"))
 			p.folder = self.custom_texturepack_path
+			
+			row = layout.row()
+			row.scale_y = 0.7
+			row.label(text=env._("Refresh MCprep data with local Minecraft install"))
+			box = layout.box()
+			split = util.layout_split(box, factor=factor_width)
+			col = split.column()
+			col.label(text=env._("Path to Minecraft versions folder"))
```

Comment:
<img width="231" alt="Screen Shot 2025-06-21 at 10 55 27 AM" src="https://github.com/user-attachments/assets/783e1136-a481-423f-9298-0fd203df3b6f" />

Gets truncated anyways, I think this will fit:

```suggestion
			col.label(text=env._("Minecraft versions"))
```